feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt

```diff
@@ -88,6 +90,30 @@ class SessionDetailFragment : DaggerFragment() {
                 uiModel.session
                     ?.let { session -> setupSessionViews(session) }
             }
+
+        binding.bottomAppBar.setOnMenuItemClickListener {
+            handleNavigation(it.itemId)
+        }
+    }
+
+    private fun handleNavigation(@IdRes itemId: Int): Boolean {
+        val navController = findNavController()
+        return try {
+            // ignore if current destination is selected
+            if (navController.currentDestination?.id == itemId) return false
+            val builder = NavOptions.Builder()
+                .setLaunchSingleTop(true)
+                .setPopUpTo(R.id.main, false)
```

Comment:
Thanks for your first-time contribution🎉
It doesn't need an option in SessionDetail to go back to this page after navigating floor map.
Could you please try removing these above two lines and check🙇‍♂️

Reply:
Thank you for the review.
Removed and pushed.